helm/drone-fork-approval-extension/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "drone-fork-approval-extension.fullname" . }}
+  labels:
+    {{- include "drone-fork-approval-extension.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "drone-fork-approval-extension.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        checksum/secret: {{ .Values.secret | sha256sum }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "drone-fork-approval-extension.selectorLabels" . | nindent 8 }}
+    spec:
```

Comment:
I think you probably want to set automountServiceAccountToken: false 

https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/

Reply:
Fixed in 92336c8.